addons/web/static/src/scss/modal.scss

```diff
@@ -90,6 +90,7 @@
                     border: none;
 
                     .modal-header {
+                        padding-right: 1rem;
```

Comment:
when do you need this? It seems to be the default value (`padding: 1rem 1rem`)

Reply:
Because of [this](https://github.com/odoo/enterprise/blob/master/web_enterprise/static/src/scss/modal_mobile.scss#L29-L34) in `web_mobile`.

I'm gonna modify that instead and do `padding: 0 1rem;` and remove the `.modal-title` margins.